sdk/servicebus/Microsoft.Azure.ServiceBus/tests/TokenProviderTests.cs

```diff
@@ -61,13 +28,32 @@ public async Task SasTokenWithLargeExpiryTimeShouldBeAccepted()
 
                 try
                 {
-                    var msg = await receiver.ReceiveAsync(TimeSpan.FromSeconds(5));
+                    var msg = await receiver.ReceiveAsync(TimeSpan.FromSeconds(5)).ConfigureAwait(false);
                 }
                 finally
                 {
-                    await receiver.CloseAsync();
+                    await receiver.CloseAsync().ConfigureAwait(false);
                 }
             });
         }
+
+        [Fact]
+        public async Task AadAuthCallback()
```

Comment:
I'd suggest we consider avoiding the abbreviation here and try to find a name that more clearly expresses the semantics of what scenario the test is attempting to validate.